scripts/amqp_listen.py

```diff
@@ -34,13 +35,13 @@ async def main() -> None:
                 async with message.process():
                     parsed_json = ast.literal_eval(message.body.decode("utf-8"))
                     # print pretty ascii separator with "message received" in the middle
-                    print("".center(80, "*"))
-                    print("".center(80, "="))
-                    print("message received".center(80, "="))
-                    print("".center(80, "="))
-                    print(json.dumps(parsed_json, indent=4))
-                    print("".center(80, "="))
-                    print("".center(80, "*"))
+                    logger.info("".center(80, "*"))
+                    logger.info("".center(80, "="))
+                    logger.info("message received".center(80, "="))
+                    logger.info("".center(80, "="))
+                    logger.info(json.dumps(parsed_json, indent=4))
+                    logger.info("".center(80, "="))
+                    logger.info("".center(80, "*"))
```

Comment:
The script is not part of the application and should not use the logging system : it's just a fake client.